bundle/manifests/ibm-common-service-operator.clusterserviceversion.yaml

```diff
@@ -8,11 +8,6 @@ metadata:
           "apiVersion": "operator.ibm.com/v3",
           "kind": "CommonService",
           "metadata": {
-            "labels": {
```

Comment:
Why are those labels removed?

Reply:
> Why are those labels removed?

The CR gets created on install, and it never has the labels, so these seem redundant.

But also, the labels are implying the CR is part of the operator instance, but they are not. CRs create applications/operands, but are not part of the CR itself. This might be a convention we'd need to discuss, but we dont label CRs as being managed-by an operator. Managed by would imply that if I change the CR, the CS operator would revert it back.

Reply:
Agreed with the above on Slack.